### PR TITLE
update the template for projcfg.ini in rt-studio

### PR DESCRIPTION
**Pull request by ErikChanGit**
拉取/合并请求描述：(PR description)
[
scons --dist-ide 生成工程时，修改工程中 projcfg.ini 模板
studio 在导入 bsp 时，调用了 scons --dist-ide 命令, 但自动生成的 projcfg.ini 不符合实际需求, 更新后方便后续维护
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

当前拉取/合并请求的状态 Intent for your PR
必须选择一项 Choose one (Mandatory):
- [ ]  本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo
代码质量 Code Quality：
我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:
 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x]  代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x]  没有垃圾代码，代码尽量精简，不包含#if 0代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x]  所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [x] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code](https://github.com/RT-Thread/rt-thread/documentation/contribution_guide/coding_style_en.txt)

### CODE REVIEW

File: tools/rt_studio.py [hunk 1]
```diff
@@ -336,16 +335,16 @@ def gen_project_file(output_file_path):
...
+											project_name=project_name,
+											output_project_path=os.path.abspath(""))
+		dir_name = os.path.dirname(output_file_path)
+		if not os.path.exists(dir_name):
+			os.makedirs(dir_name)
+		with open(output_file_path, 'w') as f:
+			f.write(w_str)
+			return True
+	except Exception as e:
+		return False
```
Comment (mysterywolf) [inline tools/rt_studio.py:350]: 请不要使用TAB 4空格代替

Comment (ErikChanGit) [reply tools/rt_studio.py:350]: 好的

**Comment by mysterywolf** [comment]
请签署CLA协议

**Comment by mysterywolf** [comment]
@Guozhanxin 郭老师请麻烦批准一下CI流程

**Comment by mysterywolf** [comment]
126fcb23e6db7b9eb141a9a93577394
CI抽风，如果PR没有问题可以忽略CLA直接合并

**Review by Guozhanxin — APPROVED** [review]

**Pull request merged**